src/v2/cookbook/dockerize-vuejs-app.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Dockerize Vue.js App
+type: cookbook
+order: 8
```

Comment:
We're getting close on this, can you please update the order to the last order so that it doesn't collide with other existing entries? I think this would be 13. Thanks!

Reply:
Fixed as per review! 😎 

I'm wondering if there is a better way to handle this...